content/source/layouts/_sidebar.erb

```diff
@@ -6,25 +6,22 @@
   </div>
 
   <ul class="nav sidebar-nav">
-    <li><a data-track='intro' href="/intro/index.html">Intro</a></li>
-    <li><a data-track='learn' href="https://learn.hashicorp.com/terraform/">Learn</a></li>
+    <li><a data-track='overview' href="/">Overview</a></li>
+    <li><a data-track='registry' href="https://registry.terraform.io">Registry</a></li>
+    <li><a data-track='learn' href="https://learn.hashicorp.com/terraform/">Tutorials</a></li>
     <li><a data-track='community' href="/community.html">Community</a></li>
-    <li><a data-track='enterprise' href="https://www.hashicorp.com/products/terraform/?utm_source=oss&utm_medium=header-nav&utm_campaign=terraform">Enterprise</a></li>
+    <li><a data-track='github' href="https://github.com/hashicorp/terraform">GitHub</a></li>
+    <li><a data-track='download-cli' href="/downloads.html">Download CLI</a></li>
+    <li><a data-track='sign-up' href="https://app.terraform.io/signup/account?utm_source=docs_banner">Terraform Cloud</a></li>
```

Comment:
I believe this should be linking to `/cloud` (my suggestion here includes changing the `data-track` attribute...I chose `cloud`, but that may not be right)

```suggestion
    <li><a data-track='cloud' href="/cloud">Terraform Cloud</a></li>
```